common/models/model-definition.js

```diff
@@ -136,7 +136,9 @@ function ready(ModelDefinition) {
         async.parallel([
           removeModelDef,
           removeModelDefJs
-        ], cb);
+        ], function(results) {
```

Comment:
This used to return an object, but @superkhau change to use async.parallel changes it to an array which is what caused the regression.